spec/unit/transaction/report_spec.rb

```diff
@@ -498,6 +528,10 @@ def add_statuses(count, type = :file)
         expect(@report.summary).to be_include(main)
       end
     end
+
+    it 'should sort total at the very end of the time metrics' do
+      expect(@report.summary).to match(/Total: \d+\.\d{2}\\nVersion:/)
```

Comment:
This test currently isn't cooperating, so I'm just fighting with white space expectations to get it passing

Reply:
Maybe need to specify multiline match `/.../m`?